app/assets/javascripts/app/views/people.coffee

```diff
@@ -0,0 +1,28 @@
+PersonView = Backbone.View.extend {
+  className: 'card'
+  template: _.template "<img class='image' src='<%= imagePath %>'></img>" +
+    "<span class='name'><%= name %></span>" +
+    "<span class='title'><%= title %></span>" +
+    "<span class='bio'><%= bio %></span>"
+  render: ->
```

Comment:
Prefer leaving a single line of whitespace between function definitions.  Combining class level keys (like `className`, `template`, etc... without gaps are fine)